src/components/Container/Container.less

```diff
@@ -0,0 +1,31 @@
+.lucid-Container {
+	overflow: hidden;
+	border-radius: 6px;
+	margin: 0 10px 10px 10px;
+	position: relative;
+	background-color: @color-white;
+	border: 1px solid @color-borderColor;
+
+	&-header {
+		padding: 12px;
```

Comment:
Prefer avoiding hard-coded values if they're repeatedly used throughout the file. I believe I've seen `12px` padding value in at least one other component before... might be worth discussing with UX whether this is a sort of standard that we can abstract into the `variables.less` file.